pages/glossary/letters-or-numbers.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Letters or numbers
+key: letters-or-numbers
```

Comment:
Where is this definition used? I couldn't find it. If this isn't used, please remove.

Reply:
Didn't realise it was still there. Will remove.